Spring-Boot/Spring-Boot-Problem/src/test/java/tw/waterball/judgegirl/springboot/problem/controllers/ProblemControllerTest.java

```diff
@@ -87,15 +90,15 @@ private void givenProblemSavedWithProvidedCodesAndTestcaseIOs() {
         testcaseIOsZip = ZipUtils.zipFilesFromResources("/stubs/in/", "/stubs/out/");
 
         this.problem = problemRepository.save(problem,
-                singletonMap(problem.getLanguageEnv(Language.C), new ByteArrayInputStream(providedCodesZip))
+                singletonMap(problem.getLanguageEnv(C), new ByteArrayInputStream(providedCodesZip))
```

Comment:
I'm think about the original one `Language.C` is more readable than `C`, but if this is recommended by others you might ignore this.